telegram/contrib/botan.py

```diff
@@ -44,3 +46,23 @@ def track(self, message, event_name='event'):
         except URLError as error:
             self.logger.warn('Botan track error ' + str(error.reason))
             return False
+
+    def shorten(self, url, user_id):
+        if not (url.startswith('http://') or url.startswith('https://')):
+            url = 'http://' + url
+        try:
+            url2 = self.shorten_template.format(token=str(self.token),
+                                                user_ids=str(user_id),
+                                                url=str(url))
+            request = Request(url2,
+                              headers={'Content-Type': 'application/json'})
```

Comment:
we allow 99 chars in a line (per pep8 extension). can you please "join" this lines? (that probably what cause yapf in travis to fail)